docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -237,6 +238,25 @@ miniziti_login() {
             --password
 }
 
+openZitiConsole() {
+    console_url="https://miniziti-console.$MINIKUBE_PROFILE.internal"
```

Comment:
What if --no-hosts?

Reply:
Excellent question! I'll get working on storing this state!

Reply:
The latest commit to this PR addresses this issue, and your comment here: 

https://github.com/openziti/ziti-doc/issues/682#issuecomment-1716620508